cookbook/pattern_matching.md

```diff
@@ -0,0 +1,86 @@
+---
+title: Pattern Matching
+---
+
+# Pattern Matching
+
+## Using the `match` keyword
+
+Like many other languages, nu offers a [`match`](https://www.nushell.sh/commands/docs/match.html#frontmatter-title-for-core) keyword. Usually this is used as a slightly more ergonomic version of `if-else` statements if you have many branches
+
+```shell
+> [black red yellow green purple blue indigo] | each {|c|
+ match $c {
+  "black" => "classy"
+  "red" | "green" | "blue" => "fundamental"
+  "yellow" | "purple" => "vibrant"
+  _ => "innovative"
+ }
+}
+───┬────────────
+ 0 │ classy
+ 1 │ funamental
+ 2 │ vibrant
+ 3 │ funamental
+ 4 │ vibrant
+ 5 │ funamental
+ 6 │ innovative
+───┴────────────
+```
+
+The equivalent in `if-else` statements would be:
+
+```shell
+> [black red yellow green purple blue] | each {|c|
+ if ($c == "black") {
+  "classy"
+  } else if ( $c in ["red", "green", "blue"]) {
+   "fundamental"
+  } else if ( $c in ['yellow', "purple"]){
+   "vibrant"
+  }  else {
+   "innovative"
+  }
+ }
+}
+```
+
+As you can see you can also use comound expressions in match statements (in this case used with `|`). Also notice the `_` case at the end, this is called the default arm and is used.
```

Comment:
did you mean *command* instead of *comound*? :yum:

Reply:
https://github.com/nushell/nushell.github.io/pull/1083/commits/32067f1dd7f61b1f008e54b401f2991ff9641f76